### DISCUSSION TITLE
"Engine did not send setup for non-standard variant"

### DISCUSSION THREAD

**Discussion by Emersonal** [fairy-stockfish/Fairy-Stockfish#313] [Q&A]
When I attempt to load the Fairy Stockfish engine set to chigorin, tencubed, or any other non-default winboard variant, I get this message. How do I set up the board for one of these variants?

**Comment by ianfab** [comment]
I can not reproduce this (on Linux using XBoard). Which versions of Winboard and Fairy-Stockfish are you using? And how does your engine command in `Engine`>`Edit Engine List...` look like?

**Reply by Emersonal** [reply]
For Fairy-stockfish I am using fairy-stockfish-largeboard_x86-64.exe from the latest release (13.1) For Winboard I am using 4.8.0. My engine command is "Fairy-SF" /variant=tencubed -fcp "C:\Users\Emerson Alden\Downloads\fairy-stockfish-largeboard_x86-64.exe"
for tencubed and "Fairy-SF" /variant=chigorin -fcp "C:\Users\Emerson Alden\Downloads\fairy-stockfish-largeboard_x86-64.exe" for chigorin. It is playing other variants just fine.

**Reply by ianfab** [reply]
I would recommend to try a recent Winboard version (4.9+), see e.g. the link in https://github.com/ianfab/Fairy-Stockfish/wiki/Graphical-user-interfaces#xboardwinboard. Otherwise your commands look good.

**Reply by Emersonal** [reply]
That worked, thank you for the advice. The link under "up-to-date WinBoard version" on the wiki is broken though.

**Reply by ianfab** [reply]
For me the link works fine (in Chrome). Does http://hgm.nubati.net/WinBoard-AA.zip not work you?

**Reply by Emersonal** [reply]
Yes, that's the one that doesn't work for me. I'm also in chrome. Its no big deal because I just downloaded Winboard 4.9.1 from a different place.
